docs/custom_promise_types/modules.md

```diff
@@ -55,6 +56,7 @@ These attributes are handled by the agent, and cannot be used inside promise mod
 * `handle`
 * `meta`
 * `with`
+* `classes`
```

Comment:
What happens if I try to make a custom promise with attribute `scope` or `promise_kept`, does it work? Based on your recent changes, I guess not, and so I think the full list of attributes which are not allowed should be listed here.

Reply:
Good point!